osf/models/node.py

```diff
@@ -2321,10 +2321,7 @@ def file_write_scope(self):
         return oauth_scopes.CoreScopes.NODE_FILE_WRITE
 
     def get_doi_client(self):
-        if settings.DATACITE_URL and settings.DATACITE_PREFIX:
-            return DataCiteClient(base_url=settings.DATACITE_URL, prefix=settings.DATACITE_PREFIX)
-        else:
-            return None
+        return DataCiteClient(base_url=settings.DATACITE_URL, prefix=settings.DATACITE_PREFIX)
```

Comment:
What happens if the DATACITE_URL or DATACITE_PREFIX aren't set? Does it result in an exception that we would catch? Or do we always have _something_ for these settings now?

Reply:
They should always be set now, just different for access prod vs. staging.